<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSettings</class>
 <widget class="QDialog" name="CameraSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblCameraMode">
         <property name="text">
          <string>Camera Mode</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbCameraMode">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Perspective</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orthogonal</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblFieldOfView">
         <property name="text">
          <string>Field of view</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="dblFieldOfView">
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>60.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblNear">
         <property name="text">
          <string>Near</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="dblNear">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000000000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblFar">
         <property name="text">
          <string>Far</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="dblFar">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000000000000000.000000000000000</double>
         </property>
         <property name="value">
          <double>1600.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dblFieldOfView</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CameraSettings</receiver>
   <slot>setFieldOfView(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>-20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dblNear</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CameraSettings</receiver>
   <slot>setNear(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dblFar</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CameraSettings</receiver>
   <slot>setFar(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dblFieldOfView</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CameraSettings</receiver>
   <slot>setFieldOfView(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>CameraSettings</receiver>
   <slot>clicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setFieldOfView(double)</slot>
  <slot>setNear(double)</slot>
  <slot>setFar(double)</slot>
  <slot>clicked(QAbstractButton*)</slot>
 </slots>
</ui>
